packages/stargate/src/aminomsgs.ts

```diff
@@ -377,7 +377,7 @@ export interface AminoMsgTransfer extends AminoMsg {
      */
     readonly timeout_height: AminoHeight;
     /**
-     * Timeout timestamp (in nanoseconds). The timeout is disabled when set to 0.
+     * Timeout timestamp in absolute nanoseconds since unix epoch. The timeout is disabled when set to 0.
```

Comment:
```suggestion
     * Timeout timestamp in nanoseconds since Unix epoch. The timeout is disabled when set to 0.
```

Reply:
Cool! I didn't know you can suggest changes. Thanks.